### DISCUSSION TITLE
Any way to install Audacious on OSX Mojave?

### DISCUSSION THREAD

**Discussion by shoebillanon** [audacious-media-player/audacious#97] [Q&A]
I'm stuck on OSX Mojave (10.14.6) since some very important drivers and software for music production would stop working on Catalina+, is there a way to install Audacious on this version of OSX? Homebrew will attempt to install moltenvk and fail.
<img width="682" alt="Screenshot 2023-12-23 at 17 45 18" src="https://github.com/audacious-media-player/audacious/assets/13932438/812c64da-c1a6-44cc-9aae-5e06036fa44b">

**Comment by radioactiveman** [comment]
`molten-vk` seems to be a dependency of [Qt 6](https://formulae.brew.sh/formula/qt) which itself is required for [Audacious](https://formulae.brew.sh/formula/audacious).
If this is the only package which cannot be compiled on macOS 10.14, you may try to use [Qt 5](https://formulae.brew.sh/formula/qt@5) instead which does not depend on molten-vk. Audacious is compatible with both Qt versions.

We can't give you further support here though and I am also not really familiar with Homebrew. Your best option is probably to see how to change the Audacious [formula](https://github.com/Homebrew/homebrew-core/blob/1f400d967445775bc446cc1f438848b665ed2098/Formula/a/audacious.rb) to use `qt@5` instead of `qt` and drop the `-Dqt6=true` statement. You may also ask for support about this via the [Homebrew discussion](https://github.com/orgs/Homebrew/discussions).

Good luck! I hope my hints were useful.

**Reply by shoebillanon** [reply]
Removing that line and changing qt to qt@5 seems to have done the trick, I spent the last 5 hours just installing dependencies and whatnot. Unfortunately something fails while installing one of the dependencies (snappy) so I didn't even get to complete the install. Oh well.
